<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>auth_page</class>
 <widget class="QMainWindow" name="auth_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>373</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Zapchat</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/icon/Zapchat-icon.png</normaloff>../res/icon/Zapchat-icon.png</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="line">
    <property name="windowModality">
     <enum>Qt::NonModal</enum>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>165</y>
      <width>500</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="reSend_sms_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Send Again</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>187</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>remain :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verify_dig4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>100</width>
      <height>149</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border-style: solid;
border-width:3px;
border-color: rgb(106, 110, 118);</string>
    </property>
    <property name="inputMask">
     <string>9</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verify_dig2">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>10</y>
      <width>100</width>
      <height>149</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border-style: solid;
border-width:3px;
border-color: rgb(106, 110, 118);</string>
    </property>
    <property name="inputMask">
     <string>9</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verify_dig1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>100</width>
      <height>149</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border-style: solid;
border-width:3px;
border-color: rgb(106, 110, 118);</string>
    </property>
    <property name="inputMask">
     <string>9</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="verify_dig3">
    <property name="geometry">
     <rect>
      <x>316</x>
      <y>10</y>
      <width>100</width>
      <height>149</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>34</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border-style: solid;
border-width:3px;
border-color: rgb(106, 110, 118);</string>
    </property>
    <property name="inputMask">
     <string>9</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="verify_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>501</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Text</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
border-style: solid;
border-width:3px;
border-color: rgb(17, 17, 255);
background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Verify</string>
    </property>
   </widget>
   <widget class="QLabel" name="back2info_link">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>324</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Sakkal Majalla</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 170, 255);
</string>
    </property>
    <property name="text">
     <string>Is your informations wrong?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Timer_lable">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>187</y>
      <width>55</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1:40</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
